supervisor/addons/model.py

```diff
@@ -614,15 +619,22 @@ def _available(self, config) -> bool:
         except (AwesomeVersionException, TypeError):
             return True
 
+    def _arch(self, config) -> str:
```

Comment:
I don't have a good reason, I was just following patterns I was seeing to keep the code consistent. I had to modify `image` as a part of its change and I saw it had separated the code into `image` and `_image` so I did the same. I meant to ask about that actually since I wasn't sure what the advantage was of doing it that way. I can certainly merge that code into `arch` though if we prefer.

Reply:
Yeah, the image is a bit special as he have to return the raw value sometimes or the lookup value